tensorflow/python/keras/initializers.py

```diff
@@ -84,6 +84,9 @@ class TruncatedNormal(TFTruncatedNormal):
     seed: A Python integer. Used to create random seeds. See
       `tf.set_random_seed` for behavior.
     dtype: The data type. Only floating point types are supported.
+
+  Returns:
+      TruncatedNormal instance.
```

Comment:
The indentation is a bit off.
```suggestion
    A TruncatedNormal instance.
```